instance/server/Client/Admin/Views.elm

```diff
@@ -98,7 +98,7 @@ swimlaneUserView user =
     a
         [ class SwimlaneUser
         -- TODO: move out into helper
-        , href <| routes.viewSingleUser ++ "?id=" ++ user.token
+        , href <| (routePath ViewSingleUser) ++ "?id=" ++ user.token
```

Comment:
Why not an hrefLink here? Because `routePath` only takes one arg? I feel like there's a better way of appending get paramaters on the end